examples/not.rst

```diff
@@ -204,7 +205,7 @@ ask for 5000 samples.
 
 >>> Q = {('x', 'x'): -1, ('x', 'z'): 2, ('z', 'x'): 0, ('z', 'z'): -1}
 >>> response = sampler.sample_qubo(Q, num_reads=5000)
->>> for sample, energy, num_occurrences in response.data():   # doctest: +SKIP
+>>> for sample, energy, num_occurrences, _chain_break_fraction in response.data():   # doctest: +SKIP
 ...    print(sample, "Energy: ", energy, "Occurrences: ", num_occurrences)
```

Comment:
For today's urgent update to virtualgraphs I also snuck in a change to update for the updated dimod.response. So now this proposed change has a conflict.